libraries/AC_Avoidance/AC_Avoid.cpp

```diff
@@ -214,13 +247,15 @@ void AC_Avoid::adjust_speed(float kP, float accel, float heading, float &speed,
 }
 
 // adjust vertical climb rate so vehicle does not break the vertical fence
-void AC_Avoid::adjust_velocity_z(float kP, float accel_cmss, float& climb_rate_cms, float dt)
+void AC_Avoid::adjust_velocity_z(float kP, float accel_cmss, float& climb_rate_cms, float& backup_speed, float dt)
 {
     // exit immediately if disabled
     if (_enabled == AC_AVOID_DISABLED) {
         return;
     }
-
+    if (!AP_AVOID_ENABLE_Z) {
```

Comment:
Perhaps we should excise the code using `#ifdef` instead

Reply:
Yes, I agree. it doesn't make any functional difference but using #ifdef would match our coding style better.